spec/Appendix B -- Grammar Summary.md

```diff
@@ -79,9 +79,7 @@ NegativeSign :: -
 NonZeroDigit :: Digit but not `0`
 
 FloatValue ::
-  - IntegerPart FractionalPart ExponentPart [lookahead != {Digit, `.`, NameStart}]
-  - IntegerPart FractionalPart [lookahead != {Digit, `.`, NameStart}]
-  - IntegerPart ExponentPart [lookahead != {Digit, `.`, NameStart}]
+  - IntegerPart FractionalPart? ExponentPart? [lookahead != {Digit, `.`, NameStart}]
```

Comment:
This edit is not equivalent; specifically it allows for only a `IntegerPart` and no `FractionalPart` nor `ExponentPart`. This causes the definition to overlap (cause ambiguity) with `IntValue`.

Reply:
Yes your are right, thanks for the review.